geo-types/src/geometry/line_string.rs

```diff
@@ -134,42 +136,48 @@ use core::ops::{Index, IndexMut};
 
 #[derive(Eq, PartialEq, Clone, Debug, Hash)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-pub struct LineString<T: CoordNum = f64>(pub Vec<Coord<T>>);
+pub struct LineString<C: geo_traits::Coord = Point<f64>>(pub Vec<C>);
 
 /// A [`Point`] iterator returned by the `points` method
-#[derive(Debug)]
-pub struct PointsIter<'a, T: CoordNum + 'a>(::core::slice::Iter<'a, Coord<T>>);
+pub struct PointsIter<'a, C: geo_traits::Coord + 'a>(::core::slice::Iter<'a, C>);
+
+impl<'a, C: fmt::Debug + geo_traits::Coord + 'a> fmt::Debug for PointsIter<'a, C> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
Do you know why we have to roll our own Debug here?

Reply:
Fixed in https://github.com/georust/geo/pull/1169/commits/559dc725c104ae87b1b994b124c7aebe7dc98de4